backend/cluster/list_resources.go

```diff
@@ -398,8 +430,9 @@ func extractKeys(resource unstructured.Unstructured, resourceType string, resour
 			}
 		}
 
-		resourceDetailsTruncated.Keys = fmt.Sprintf("%v", keys)
+		sort.Strings(keys)
```

Comment:
Does it need to be sorted?

Reply:
*general comment

Reply:
Yes and no. Keys in map are non-deterministic. They change, which is probably not good in general, but trigers errors in tests quite often.